packages/vite-plugin-cloudflare/e2e/fixtures/basic/package.json

```diff
@@ -4,10 +4,10 @@
 	"private": true,
 	"type": "module",
 	"scripts": {
-		"build": "tsc -b && vite build",
+		"build": "vite build",
 		"dev": "vite",
 		"lint": "eslint .",
-		"preview": "vite preview"
+		"preview": "vite build && vite preview"
```

Comment:
nit: I wonder if it'd be cleaner to have a `build` script, a `preview` script and a `buildAndPreview` script (and replace `preview` in `commands` with `buildAndPreview`) 🤔

Reply:
OK